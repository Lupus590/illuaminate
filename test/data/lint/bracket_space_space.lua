-- config: (lint (only format:bracket-space) (bracket-spaces (table space)))

local _ = { 1}, {1 }, {1}, { 1 }
