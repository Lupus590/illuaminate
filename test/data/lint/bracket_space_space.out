bracket_space_space.lua:[3:11-3:14]: Inconsistent spacing [format:bracket-space]
Should have spaces within the brackets.
   │
 3 │ local _ = { 1}, {1 }, {1}, { 1 }
   │           ^^^^
bracket_space_space.lua:[3:17-3:20]: Inconsistent spacing [format:bracket-space]
Should have spaces within the brackets.
   │
 3 │ local _ = { 1}, {1 }, {1}, { 1 }
   │                 ^^^^
bracket_space_space.lua:[3:23-3:25]: Inconsistent spacing [format:bracket-space]
Should have spaces within the brackets.
   │
 3 │ local _ = { 1}, {1 }, {1}, { 1 }
   │                       ^^^
No errors and 3 warnings
 -- config: (lint (only format:bracket-space) (bracket-spaces (table space)))
 
-local _ = { 1}, {1 }, {1}, { 1 }
+local _ = { 1 }, { 1 }, { 1 }, { 1 }
 
